### PR TITLE
Fix/text color mobile

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: examples/main/shared/editor/Editor.jsx [hunk 1]
```diff
@@ -55,10 +55,14 @@ export default class Editor extends PureComponent {
     };
     this.helpers = {
       //these are for testing purposes only
-      onPluginAdd: async (plugin_id, entry_point, version) => console.log('biPluginAdd', plugin_id, entry_point, version),
-      onPluginDelete: async (plugin_id, version) => console.log('biPluginDelete', plugin_id, version),
-      onPluginChange: async (plugin_id, changeObj, version) => console.log('biPluginChange', plugin_id, changeObj, version),
-      onPublish: async (postid, callback, version) => console.log('biOnPublish', ({ data }) => data, version),
+      onPluginAdd: async (plugin_id, entry_point, version) =>
```
Comment (shaulgo) [inline examples/main/shared/editor/Editor.jsx]: is this only whitespace changes?

File: packages/plugin-text-color/web/src/toolbar/BaseTextColor.jsx [hunk 2]
```diff
@@ -38,9 +38,8 @@ export default class BaseTextColor extends Component {
   };
 
   closePanel = editorState => {
-    this.setState({ isPanelOpen: false });
+    this.setState({ isPanelOpen: false }, () => this.preserveSelectionState(editorState));
```
Comment (shaulgo) [inline packages/plugin-text-color/web/src/toolbar/BaseTextColor.jsx]: do you know what this does?

File: examples/main/shared/editor/Editor.jsx (hunk 1 above)
Comment (talevy17) [reply examples/main/shared/editor/Editor.jsx]: Yes

File: packages/plugin-text-color/web/src/toolbar/BaseTextColor.jsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-text-color/web/src/toolbar/BaseTextColor.jsx]: adds the next function call (that sets the editor state) as a callback to the state change that closes the modal
I'm not sure if the forceselection is needed though it's probably there for a reason

**Pull request merged**